sw/airborne/modules/computer_vision/detect_gate.c

```diff
@@ -128,9 +119,19 @@ volatile float detect_gate_z;
 
 static pthread_mutex_t gate_detect_mutex;            ///< Mutex lock fo thread safety
 
+// Module variables
+struct vision_relative_position_struct {
```

Comment:
Seems a little redundant to name the struct struct, I think the convention is to use _t